oic.baseResource/oic.baseResource.json

```diff
@@ -0,0 +1,61 @@
+{
+    "id": "http://openinterconnect.org/iotdatamodels/schemas/oic.baseResource.json#",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "description": "Copyright (c) 2016 Open Connectivity Foundation, Inc. All rights reserved.",
+    "title": "Base Resource",
+    "definitions": {
+        "oic.r.baseresource": {
```

Comment:
>oic.r.baseresource [](start = 9, length = 18)

There is actually a "baseResource" RAML at oneiota.org, should that be included here?

(I didn't find a "core" RAML.)